mentoring/templates/html/mcqblock_rating.html

```diff
@@ -6,35 +6,55 @@ <h3 class="question-title">QUESTION {% if not self.lonely_step %}{{ self.step_nu
     <div class="choice">
       <div class="choice-result fa icon-2x"></div>
-      <label><input class="choice-selector" type="radio" name="{{ self.name }}" value="1"{% if self.student_choice == '1' %} checked{% endif %} />1</label>
-      <span class="low"> - {{ self.low }}</span>
+      <div class="choice-selector">
+        <input id="choice-{{ self.uuid }}-1" type="radio" name="{{ self.name }}" value="1"{% if self.student_choice == '1' %} checked{% endif %} />
+      </div>
+      <label class="choice-label" for="choice-{{ self.uuid }}-1">
+        1 <span class="low"> - {{ self.low }}</span>
+      </label>
       <div class="choice-tips"></div>
     </div>
     <div class="choice">
       <div class="choice-result fa icon-2x"></div>
-      <label><input class="choice-selector" type="radio" name="{{ self.name }}" value="2"{% if self.student_choice == '2' %} checked{% endif %} />2</label>
+      <div class="choice-selector">
+        <input id="choice-{{ self.uuid }}-2" type="radio" name="{{ self.name }}" value="2"{% if self.student_choice == '2' %} checked{% endif %} />
+      </div>
+      <label class="choice-label" for="choice-{{ self.uuid }}-2">2</label>
       <div class="choice-tips"></div>
     </div>
     <div class="choice">
       <div class="choice-result fa icon-2x"></div>
-      <label><input class="choice-selector" type="radio" name="{{ self.name }}" value="3"{% if self.student_choice == '3' %} checked{% endif %} />3</label>
+      <div class="choice-selector">
+        <input id="choice-{{ self.uuid }}-3" type="radio" name="{{ self.name }}" value="3"{% if self.student_choice == '3' %} checked{% endif %} />
+      </div>
+      <label class="choice-label" for="choice-{{ self.uuid }}-3">3</label>
       <div class="choice-tips"></div>
     </div>
     <div class="choice">
       <div class="choice-result fa icon-2x"></div>
-      <label><input class="choice-selector" type="radio" name="{{ self.name }}" value="4"{% if self.student_choice == '4' %} checked{% endif %} />4</label>
+      <div class="choice-selector">
+        <input id="choice-{{ self.uuid }}-4" type="radio" name="{{ self.name }}" value="4"{% if self.student_choice == '4' %} checked{% endif %} />
+      </div>
+      <label class="choice-label" for="choice-{{ self.uuid }}-4">4</label>
       <div class="choice-tips"></div>
     </div>
     <div class="choice">
       <div class="choice-result fa icon-2x"></div>
-      <label><input class="choice-selector" type="radio" name="{{ self.name }}" value="5"{% if self.student_choice == '5' %} checked{% endif %} />5</label>
-      <span class="low"> - {{ self.high }}</span>
+      <div class="choice-selector">
+        <input id="choice-{{ self.uuid }}-5" type="radio" name="{{ self.name }}" value="5"{% if self.student_choice == '5' %} checked{% endif %} />
+      </div>
+      <label class="choice-label" for="choice-{{ self.uuid }}-5">
+        5 <span class="low"> - {{ self.high }}</span>
+      </label>
```

Comment:
@mtyaka this is a bit out of the scope of the bug you're fixing, but couldn't these five almost identical groups be rendered using a loop? There's differences for 1 and 5 which can be addressed with simple `if`, and other than that these lines look quite identical.